openedx/core/djangoapps/organization_api/api/views.py

```diff
@@ -0,0 +1,39 @@
+"""
+Organizations API views.
+"""
+from rest_framework import permissions
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.response import Response
+from rest_framework.status import HTTP_404_NOT_FOUND
+from rest_framework.views import APIView
+from rest_framework_jwt.authentication import JSONWebTokenAuthentication
+from rest_framework_oauth.authentication import OAuth2Authentication
+
+from util.organizations_helpers import get_organization_by_short_name
+
+
+class OrganizationsView(APIView):
```

Comment:
It would be preferable to see all of the organization-related code in `edx-organizations`. Having the code half in one location and half in another is weird.